iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DeviceTwinCommon.java

```diff
@@ -146,7 +145,6 @@ public static Collection inputs() throws Exception
     protected static RegistryManager registryManager;
     protected InternalClient internalClient;
     protected static RawTwinQuery scRawTwinQueryClient;
```

Comment:
Since you are moving this one to test instance. Are others here not used as a common resource?

Reply:
I can move these to be per-test instance, sure